src/os/xdg.zig

```diff
@@ -193,3 +193,87 @@ test parseTerminalExec {
         try testing.expectEqualSlices([*:0]const u8, actual, &.{ "a", "-e", "b", "c" });
     }
 }
+
+/// https://specifications.freedesktop.org/basedir-spec/latest/
+pub const Dirs = enum {
+    config,
+    data,
+
+    pub fn key(self: Dirs) []const u8 {
+        return switch (self) {
+            .config => "XDG_CONFIG_DIRS",
+            .data => "XDG_DATA_DIRS",
+        };
+    }
+
+    pub fn default(self: Dirs) []const u8 {
+        return switch (self) {
+            .config => "/etc/xdg",
+            .data => "/usr/local/share:/usr/share",
+        };
+    }
+};
+
+pub const config_dirs = Dirs.config;
+pub const data_dirs = Dirs.data;
```

Comment:
I don't think you should export these like these. Callers should fully qualify it.

Reply:
Removed.

Reply:
Removed.